addons/mail/wizard/mail_compose_message.py

```diff
@@ -195,6 +196,42 @@ def _compute_body(self):
             elif not composer.template_id or not composer.body:
                 composer.body = False
 
+    @api.depends('composition_mode', 'model', 'res_ids', 'template_id')
+    def _compute_attachment_ids(self):
+        """ Attachments computation.
+        With template
+          * mass mail: take IDs from template
+        """
+        for composer in self:
+            res_ids = composer._parse_res_ids(composer.res_ids)
+            if composer.composition_mode == 'mass_mail' and composer.template_id and composer.template_id.attachment_ids:
```

Comment:
do we need to check `composer.template_id` ? 
(`composer.template_id.attachment_ids` will return False anyway if there's no template)